content/admin/org_settings/integrations/Agile-central.md

```diff
@@ -0,0 +1,83 @@
+<!--
+title: "CA Agile Central Integration"
+description: "Integrate CA Agile Central with Contrast"
+tags: "Admin organization settings integrations CA agile central"
+-->
+
+
+Integrate Agile Central with Contrast to automatically track vulnerabilities in your applications.
+
+## Prerequisites
+
+* Agile Central account URL
+* Permission to create issues in the target project
+* A running Agile Central instance accessible via HTTP to the Contrast UI
+* A project to associate the application instrumented by Contrast
+
+## Add a Configuration 
+
+To connect your Contrast organization with Agile Central, complete the following steps. 
+
+* Go to the **user menu > Organization Settings > Integrations** page, and click **Connect** in the Agile Central row. 
+* In the **Connect with Agile Central** form, add the name for the bugtracker entry, as well as the **URL** and **API Key** in the given fields. <!-- The Agile Central URL must be accessible from the Contrast UI instance being configured. -->
+
+> **Notes:** 
+* To find your Agile Central API key, log in to the Agile Central Application manager <!-- (add hyperlink) --> and go to the **API Keys** tab.  
+* Contrast saves the username, password and Agile Central URL entered in your configuration as a set of credentials. See the **Credential Sets** section for more details.
+
+* Once you complete the fields, click the button to **Test connection**. The test verifies that Contrast can reach the Agile Central instance and that the specified user can log in.
+
+<a href="assets/images/Agile-central-connect.png" rel="lightbox" title="Configure a new Agile Central integration"><img class="thumbnail" src="assets/images/Agile-central-connect.png"/></a>
+
```

Comment:
This looks slightly different than the design review. If so, we should to update to the latest.